src/main/kotlin/no/nav/arbeidsgiver/altinnrettigheter/proxy/klient/AltinnrettigheterProxyKlient.kt

```diff
@@ -21,15 +21,71 @@ class AltinnrettigheterProxyKlient(
         private val config: AltinnrettigheterProxyKlientConfig
 ) {
 
+    /**
+     * Hent alle organisasjoner i Altinn en bruker har rettigheter i.
+     *  @param selvbetjeningToken - Selvbetjening token til innlogget bruker
+     *  @param subject - Fødselsnummer til innlogget bruker (fallback funksjon)
+     *  @param filterPaaAktiveOrganisasjoner - Aktiver filtering på både Status og Type
+     *
+     *  @return en liste av alle organisasjoner
+     *   - med Status: 'Active' og Type: 'Enterprise' | 'Business', når filterPaaAktiveOrganisasjoner er 'true'
+     *   - med Status: 'Active' | 'Inactive' og Type: 'Enterprise' | 'Business' | 'Person', når filterPaaAktiveOrganisasjoner er 'false'
+     */
     fun hentOrganisasjoner(
+            selvbetjeningToken: SelvbetjeningToken,
+            subject: Subject,
+            filterPaaAktiveOrganisasjoner: Boolean
+    ): List<AltinnReportee> {
+        return hentOrganisasjonerMedEllerUtenRettigheter(
+                selvbetjeningToken,
+                subject,
+                null,
+                null,
+                filterPaaAktiveOrganisasjoner
+        )
+    }
+
+    /**
+     * Hent alle organisasjoner i Altinn en bruker har enkel rettighet i
+     *  @param selvbetjeningToken - Selvbetjening token til innlogget bruker
+     *  @param subject - Fødselsnummer til innlogget bruker (fallback funksjon)
+     *  @param serviceCode - Kode for rettigheter brukeren har for en organisasjon (henger sammen med ServiceEdition)
+     *  @param serviceEdition
+     *  @param filterPaaAktiveOrganisasjoner - Aktiver filtering på både Status og Type
+     *
+     *  @return en liste av alle organisasjoner
+     *   - med Status: 'Active' og Type: 'Enterprise' | 'Business', når filterPaaAktiveOrganisasjoner er 'true'
+     *   - med Status: 'Active' | 'Inactive' og Type: 'Enterprise' | 'Business' | 'Person', når filterPaaAktiveOrganisasjoner er 'false'
+     */
+    fun hentOrganisasjonerBasertPåRettigheter(
             selvbetjeningToken: SelvbetjeningToken,
             subject: Subject,
             serviceCode: ServiceCode,
-            serviceEdition: ServiceEdition
+            serviceEdition: ServiceEdition,
+            filterPaaAktiveOrganisasjoner: Boolean
+    ): List<AltinnReportee> {
+        return hentOrganisasjonerMedEllerUtenRettigheter(
+                selvbetjeningToken,
+                subject,
+                serviceCode,
+                serviceEdition,
+                filterPaaAktiveOrganisasjoner
+        )
+    }
+
+
+    private fun hentOrganisasjonerMedEllerUtenRettigheter(
```

Comment:
"hent organisasjoner med eller uten rettigheter" --> høres ut som om man henter alle organisasjoner, uavhengig om brukeren har rettigheter til dem eller ikke :P Kan den ikke bare hete "hentOrganisasjoner", og så snakker argumentene for seg selv?

Reply:
enig 👍 
Testet det fra en konsument, fungerte fint med forslag fra IDE